errbot/config-template.py

```diff
@@ -31,7 +31,16 @@
 # locally before publishing it. Note that you can specify only a single
 # directory, however you are free to create subdirectories with multiple
 # plugins inside this directory.
-BOT_EXTRA_PLUGIN_DIR = None 
+BOT_EXTRA_PLUGIN_DIR = None
+
+# If True, when you err wants to start a plugin and a dependency from
+# requirements.txt of the plugin is missing, it will automatically invoke
+# pip install --user for the missing dependencies.
+# It will detect if the bot is running in a virtualenv and if yes, it
+# will install it in the virtualenv instead of on the user.
+#
+# Note: be sure to give the proper autorizations to the user running the bot.
+AUTOINSTALL_DEPS = True
```

Comment:
Maybe also put a `#` in front of `AUTOINSTALL_DEPS` to comment it out, so that people upgrading can continue using their old `config.py` and do not have to add *AUTOINSTALL_DEPS* before their bot starts again (Err checks for missing configuration items and refuses to start if any are missing).